clifford/test/test_clifford.py

```diff
@@ -60,7 +60,7 @@ def test_inverse(self, algebra):
             1 / a
         for i in range(10):
             a = randomMV(layout, grades=[0, 1])
-            denominator = float(a(1)**2-a(0)**2)
+            denominator = (a(1)**2)[()]-(a(0)**2)[()]
```

Comment:
This is just cleanup, right?

Reply:
yep, the float(multivector) syntax never sits quite right with me

Reply:
Marginally shorter as:

```suggestion
            denominator = (a(1)**2)[()]-(a[()]**2)
```